types/multistream/multistream-tests.ts

```diff
@@ -0,0 +1,40 @@
+import * as fs from 'fs';
+import * as stream from 'stream';
+
+import * as through from 'through';
+import * as through2 from 'through2';
+import * as multistream from 'multistream';
```

Comment:
please use `import multistream = require('multistream')` for commonjs modules, which use `export = multistream` syntax.

Reply:
Oh, sorry, I thought that this recommendation applies only to packages that return only one function. Now I understand that it's applies to all packages that returns function.

Reply:
Fixed.